shardingsphere-infra/shardingsphere-infra-common/src/test/java/org/apache/shardingsphere/infra/database/impl/DataSourceGeneratedDatabaseConfigurationTest.java

```diff
@@ -34,32 +26,60 @@
     
     @Test
-    public void assertDataSourceGeneratedDatabaseConfiguration() {
-        String url = "jdbc:mysql://192.168.0.1:3306/foo_ds";
-        String username = "root";
-        String password = "root";
-        ConnectionConfiguration configuration = new ConnectionConfiguration(url, username, password);
-        PoolConfiguration poolConfiguration = new PoolConfiguration(2000L, 1000L, 1000L, 6, 2, false, new Properties());
-        DataSourceConfiguration dataSourceConfiguration = new DataSourceConfiguration(configuration, poolConfiguration);
-        Map<String, DataSourceConfiguration> dataSources = new HashMap<>(1);
-        dataSources.put("dsc", dataSourceConfiguration);
-        DataSourceGeneratedDatabaseConfiguration databaseConfiguration = new DataSourceGeneratedDatabaseConfiguration(dataSources,
-                Collections.singletonList(new FixtureRuleConfiguration("rule0")));
-        DataSource dataSource = databaseConfiguration.getDataSources().get("dsc");
-        assertThat(dataSource, instanceOf(HikariDataSource.class));
-        assertThat(((HikariDataSource) dataSource).getUsername(), is(username));
-        assertThat(((HikariDataSource) dataSource).getPassword(), is(password));
-        assertThat(((HikariDataSource) dataSource).getJdbcUrl(), is(url));
-        FixtureRuleConfiguration next =
-                (FixtureRuleConfiguration) databaseConfiguration.getRuleConfigurations().iterator().next();
-        assertThat(next.getName(), is("rule0"));
-        DataSourceProperties properties = databaseConfiguration.getDataSourceProperties().get("dsc");
-        assertThat(properties.getPoolPropertySynonyms().getStandardPropertyKeys().size(), is(6));
-        assertThat(properties.getConnectionPropertySynonyms().getStandardProperties().get("url"), is(url));
-        assertThat(properties.getConnectionPropertySynonyms().getStandardProperties().get("username"), is(username));
-        assertThat(properties.getConnectionPropertySynonyms().getStandardProperties().get("password"), is(password));
+    public void assertDataSource() {
+        DataSourceGeneratedDatabaseConfiguration databaseConfig = createDataSourceGeneratedDatabaseConfiguration();
+        HikariDataSource hikariDataSource = (HikariDataSource) databaseConfig.getDataSources().get("normal_db");
+        assertThat(hikariDataSource.getJdbcUrl(), is("jdbc:mock://127.0.0.1/normal_db"));
+        assertThat(hikariDataSource.getUsername(), is("root"));
+        assertThat(hikariDataSource.getPassword(), is(""));
+    }
+    
+    @Test
+    public void assertRuleConfiguration() {
+        DataSourceGeneratedDatabaseConfiguration databaseConfig = createDataSourceGeneratedDatabaseConfiguration();
+        FixtureRuleConfiguration ruleConfig = (FixtureRuleConfiguration) databaseConfig.getRuleConfigurations().iterator().next();
+        assertThat(ruleConfig.getName(), is("test_rule"));
+    }
+    
+    @Test
+    public void assertDataSourceProperties() {
```

Comment:
It is better to rename method as `assertGetDataSourceProperties`

Reply:
Done, please review again, thank you.